README.md

```diff
@@ -83,6 +84,42 @@ GLHF! <3
 
 GLHF! <3
 
+## Tantangan
+
+Anda mempunyai satu bulan penuh pada bulan November untuk membuat sebuah permainan *bebas* dengan bertemakan **peretasan, modifikasi dan/atau penggandaan**.
+
+Apa yang kita maksud dengan **bebas** berdasarkan pada peretasan, modifikasi dan/atau penggandaan? Kami sebenarnya bermaksud, *bebas* yang disarankan. Beberapa contoh mungkin seperti ini:
+
+* seorang pelari yang tak berujung dimana anda *retas* pohon biner menurun di jalur anda dengan kapak yang berpiksel,
+* sebuah pengambilan secara *modern* pada sesuatu yang klasik misalnya sebuah roguelike yang diatur di dalam dunia 3D atau dunia VR, atau
+* sebuah *penggandaan* permainan yang nyata membawa gabungan gurita/kucing kedalam dunia nyata.
```

Comment:
Maybe we should refer the original term, also the [Wikipedia indonesia](https://id.wikipedia.org/wiki/Realitas_tertambah) translate Augmented reality to "Realitas tertambah".
So ` Sebuah permainan realitas tertambah (*augmented reality*) yang membawa gurita/kucing...`